HOW/social/process/_requirements.md

```diff
@@ -0,0 +1,34 @@
+---
+Status: <DRAFT|REVIEW|REJECTED|SUPERSEDED|DEFERRED|LIVE>
```

Comment:
What happened to Holochain Improvement Proposals?

If we used that term then I'd like to propose that we rename "superseded" to "replaced" so that we can have "HIP replacements" :)

Reply:
@zippy preferred the term "how" and I don't feel strongly.